### DISCUSSION TITLE
Dudas generadas en la reunión de avances de diagramas. 26-10-2024

### DISCUSSION THREAD

**Discussion by TheMoys** [TheMoys/24-25-IdSw1-SDR#15] [Q&A] [answered]
A continuación se detallan las dudas generadas para posteriormente discutirlas con Alicia:

- Se pueden enviar diferentes tipos de documento en un mismo mensaje? Ej.: un documento de notificación al mismo tiempo que un documento que requiera confirmación y firma. 

- El mensaje define el tipo de documento que tendrá? Es decir, que si el mensaje es de tipo notificación o los documentos adjuntos pueden ser de diferentes tipos.

- Se puede rechazar un documento? Como un burofax

- Se contemplarán procesos físicos o únicamente digitales?

**Comment by TheMoys** [comment]
La reunión con Alicia queda pendiente de confirmar... El día lunes le haré llegar un correo para poder cuadrar una reunión.

**Reply by TheMoys** [reply]
La reunión con Alicia será el día martes 29 a las 9:00 am en el aula 0.10

**Comment by ghost** [comment]
<img width="490" alt="Captura de pantalla 2024-10-29 a la(s) 9 14 37" src="https://github.com/user-attachments/assets/0da55211-0580-4bd3-9e67-1497d25ba339">

**Comment by TheMoys** [comment, marked as answer]
**Apuntes tomados en la sesión de reunión 29-10-2024**

- El mensaje contiene los tipos de envió, si es de info, si es de confirmación o firma

- No esta contemplada la opción de rechazar un documento, la gestion de ese tipo de incidencias se hará fuera del sistema.
 
- La aplicación únicamente generara los documentos de resumen de los hitos del usuario con el documento.
 
- HITOS DE FECHA: enviado, accedido, confirmar recibido si es solicitado, confirmado lectura si es solicitado
 
- CATEGORIAS: PDI, PAS, Profesores asociados, proveedores, relaciones institucionales, etc. están abiertas a extensiones configurables (listas de receptores)
